src/screens/SUE/SueReportScreen.tsx

```diff
@@ -172,11 +172,19 @@ export const SueReportScreen = ({
     setIsLoading(true);
     mutateAsync(formData)
       .then(() => {
+        if (sueReportData?.images?.length > 0) {
+          setIsDone(true);
+          resetStoredValues();
+          setIsLoading(false);
+
+          return;
+        }
+
         setTimeout(() => {
           setIsDone(true);
           resetStoredValues();
           setIsLoading(false);
-        }, 1500);
+        }, 3000);
```

Comment:
what about avoiding the duplicate content of methods called when you add a condition of the time based on existing images? 3000 if no images and 0 if images, something like that?

Reply:
Definitely makes sense. I already found a bug in the code. If the image was not selected, I had to do `JSON.parse` first because `sueReport?.images?.length` was `'[]'` by default. I added this to the code and updated it.

https://github.com/smart-village-solutions/smart-village-app-app/commit/ce923a8b752c029c400c3a80046d55a917338232